### ISSUE TITLE
how should I know the header for the minimal output 

### ISSUE THREAD

**Issue by wangyihanlarry** [axboe/fio#1226]
Hi experts, 

I am confused on the fio minimal output in below, where can I get the meaning for each field?

3;fio-3.7;bio;0;0;28709340;159349;500;180166;2;1078053;720.835412;8307.158883;743;1632665;127194.504051;124259.989230;1.000000%=10158;5.000000%=24510;10.000000%=34340;20.000000%=51642;30.000000%=66846;40.000000%=80216;50.000000%=94896;60.000000%=112721;70.000000%=135266;80.000000%=173015;90.000000%=250609;95.000000%=337641;99.000000%=549453;99.500000%=1098907;99.900000%=1249902;99.950000%=1367343;99.990000%=1501560;0%=0;0%=0;0%=0;750;1632669;127915.442197;124747.192350;8656;335132;100.000000%;163034.463068;69585.644925;28707618;159339;498;180166;2;1079330;728.166029;6653.737678;1118;1592279;127550.672844;126465.260968;1.000000%=10420;5.000000%=24248;10.000000%=34340;20.000000%=51118;30.000000%=66322;40.000000%=80216;50.000000%=94896;60.000000%=112721;70.000000%=135266;80.000000%=173015;90.000000%=254803;95.000000%=337641;99.000000%=549453;99.500000%=1115684;99.900000%=1233125;99.950000%=1367343;99.990000%=1501560;0%=0;0%=0;0%=0;1132;1613624;128278.943469;126831.663948;7357;325355;100.000000%;162984.312500;69065.900861;1.618516%;1.565232%;56137;0;32798;0.1%;0.1%;0.1%;0.1%;0.1%;0.1%;100.0%;0.00%;0.00%;0.00%;0.00%;0.00%;0.00%;0.00%;0.00%;0.01%;0.01%;0.03%;0.19%;0.73%;2.57%;15.68%;33.96%;36.56%;8.92%;0.77%;0.02%;0.00%;0.00%

**Comment by axboe** [comment]
It’s described in the included documentation.

**Comment by wangyihanlarry** [comment]
I got you, my only suggestion is to add the header for the terser output as well, it is hardly to know the meaning for each filed, especially the amount also changes.

**Comment by akarasulu** [comment]
Yes, this is getting worse. There are different versions. I found it for version 3, but not the latest version 5.

**Comment by vincentkfu** [comment]
@akarasulu v5 adds min/max/mean/etc values for the BW and IOPS samples that fio periodically collects during a run. This is documented in the last lines of the `READ status` and `WRITE status` sections of https://github.com/axboe/fio/blob/master/HOWTO.rst#terse-output

v4 adds trim statistics to the v3 output.

I'd be happy to review a patch improving the documentation to make these differences more obvious.
